config-model/src/main/java/com/yahoo/vespa/model/container/http/FilterBinding.java

```diff
@@ -11,16 +11,20 @@
  */
 public class FilterBinding {
 
+    public enum Type {REQUEST, RESPONSE}
+
+    private final Type type;
     private final ComponentSpecification chainId;
     private final BindingPattern binding;
 
-    private FilterBinding(ComponentSpecification chainId, BindingPattern binding) {
+    private FilterBinding(Type type, ComponentSpecification chainId, BindingPattern binding) {
+        this.type = type;
         this.chainId = chainId;
         this.binding = binding;
     }
 
-    public static FilterBinding create(ComponentSpecification chainId, BindingPattern binding) {
-        return new FilterBinding(chainId, binding);
+    public static FilterBinding create(Type type, ComponentSpecification chainId, BindingPattern binding) {
```

Comment:
I assume this code isn't called by older versions of code, in the config server ... ?

Reply:
Nope, this method is only used inside `config-model` module.